resources/Field.php

```diff
@@ -47,27 +47,31 @@ public function __construct()
 	public function render_field_settings($field)
 	{
 		// Render a field settings that will tell us if an empty field is allowed or not
+		$returnFormatChoices = [
+			'post_object' => __('Form Object', ACF_GF_FIELD_TEXTDOMAIN),
+			'id'          => __('Form ID', ACF_GF_FIELD_TEXTDOMAIN),
+		];
+
 		acf_render_field_setting($field, [
 			'label'        => __('Return Value', 'acf'),
 			'instructions' => __('Specify the returned value on front end', 'acf'),
 			'type'         => 'radio',
 			'name'         => 'return_format',
 			'layout'       => 'horizontal',
-			'choices'      => [
-				'post_object' => __('Form Object', ACF_GF_FIELD_TEXTDOMAIN),
-				'id'          => __('Form ID', ACF_GF_FIELD_TEXTDOMAIN)
-			],
+			'choices'      => $returnFormatChoices,
```

Comment:
I've not used Code Climate and I'm curious about some of the things you've changed.

Was this a warning it raised? I think I prefer the original version. Is there a reason you shouldn't nest array initialisation like this?